### PR TITLE
Host and Seed model torrents using Torf and Transmission

### LINKED ISSUE #1: Self host Models weights using Torrent
### Description

We need to self-host the weights using Torrent on a Hetzener machine. The steps are the following:

1. Run the `transmission` daemon using `docker-compose`
2. Download the weights from HF
3. Create a `.torrent` file for each folder containing the weights.
4. Seed the torrents using the folder used for dumping the weights.

We want to seed the following models:

- [ ] premai-io/CodeLlama-34b-Instruct-hf
- [ ] petals-team/StableBeluga2
- [ ] tiiuae/falcon-180B-chat

### Missing 

Most of the logic and the scripts are in the repo. The entire flow must be tested and fixed accordingly.

### PR DESCRIPTION
**Pull request by biswaroop1547**
Currently transmission client webui can be accessed through - http://128.140.126.131:9091 (has auth), which's currently hosting and seeding.

Webseed url can be accessed through - http://128.140.126.131:8989/ (public)

Takes care of
- [premai-io/CodeLlama-34b-Instruct-hf](https://huggingface.co/premai-io/CodeLlama-34b-Instruct-hf)
- [petals-team/StableBeluga2](https://huggingface.co/petals-team/StableBeluga2)

Note: Need to support [tiiuae/falcon-180B-chat](https://huggingface.co/tiiuae/falcon-180B-chat) later, as currently not supporting due to memory constraints.

closes #1

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,9 +7,13 @@ services:
       - PUID=1000
       - PGID=1000
       - TZ=Etc/UTC
+      - USER=prem
+      - FILE__PASS=/transmission_password
     volumes:
       - data-config:/config
-      - data-downloads:/downloads
+      - /mnt/HC_Volume_37156271/from-hf-to-torrent/data:/downloads
```
Comment (filopedraz) [inline docker-compose.yml]: Can't we use docker volumes? and tell to Docker to access the external volume?

Comment (biswaroop1547) [reply docker-compose.yml]: sure, I'll update

Comment (biswaroop1547) [reply docker-compose.yml]: Using external volume would create a requirement before running this compose file that the volume should already be created and present. Instead we can create and mount these volumes at runtime of compose file (using type `local`), that sounds fine?

Comment (filopedraz) [reply docker-compose.yml]: Yep

**Review by filopedraz — APPROVED** [review]

**Comment by filopedraz** [comment]
LGTM

**Pull request merged**